mailpoet/assets/js/src/analytics_event.js

```diff
@@ -38,12 +37,12 @@ function track(name, data = []) {
 }
 
 function exportMixpanel() {
-  MailPoet.forceTrackEvent = track;
+  window.MailPoet.forceTrackEvent = track;
 
-  if (window.mailpoet_analytics_enabled && MailPoet.libs3rdPartyEnabled) {
-    MailPoet.trackEvent = track;
+  if (window.mailpoet_analytics_enabled && window.MailPoet.libs3rdPartyEnabled) {
+    window.MailPoet.trackEvent = track;
   } else {
-    MailPoet.trackEvent = function emptyFunction() {};
+    window.MailPoet.trackEvent = function emptyFunction() {};
```

Comment:
The code was using imports `import MailPoet from 'mailpoet'` which was misleading, I thought making it more explicit would allow us to see it better in future, it's already more visible since you caught it ‌😀

Reply:
Right, if it works, I'm fine with it. The current (existing) behavior in this file is really messy but of course, it's not the point of this task to refactor it.